docs/devGuide/development/workflow.md

```diff
@@ -126,6 +126,22 @@ The sections below has more information about various stages of submitting a PR.
 
   </modal>
 
+#### Keeping your fork up to date
+
+When new PRs are merged into the `master` branch, your `master` branch will be out of sync with the main repository. One way to update your branches to branch from the latest master is as follows:
+
+1. Go to your fork on Github and click 'Sync Fork'
+
+1. Checkout your `master` branch and `git pull`
+
+1. Checkout your feature branch and [rebase](https://git-scm.com/book/en/v2/Git-Branching-Rebasing) the branch or [merge](https://git-scm.com/docs/git-merge) `master` into the branch 
+
+1. Fix any merge conflicts (if applicable)
+
+<box type="tip" seamless>
+You may want to use a tool like [GitTown](https://www.git-town.com/) to speed up the process.
```

Comment:
```suggestion

You may want to use a tool like [GitTown](https://www.git-town.com/) to speed up the process.
```

markdown link not working likely due to missing space:
![image](https://github.com/MarkBind/markbind/assets/41845017/2ce4cdd9-ceef-4f56-8a02-3eaba8d79de2)